### PR TITLE
Cryptography Util Examples

### PR DESCRIPTION
**Pull request by Luzzotica**
## Reason:

There were no clear examples on how to use cryptography utils, so I added the simplest, and most commonly used examples.

## Changes made (preferably with images/screenshots):

Screen Shot 2023-02-06 at 3 50 36 PM

## Check off the following:

- [x] I have reviewed my changes and run the appropriate tests.
- [x] I have have run `rush change` to add the appropriate change logs.
- [x] I have added/edited docs.
- [x] I have added tutorials.
- [x] I have double checked and DEFINITELY added docs.

### CODE REVIEW

**Review by Randynamic — APPROVED** [review]

**Comment by Randynamic** [comment]
@Luzzotica Thx for the contribution!!!

**Review by ash-vd — APPROVED** [review]

File: packages/libs/cryptography-utils/README.md [hunk 1]
```diff
@@ -7,11 +7,39 @@
...
+
+// Create a command
+let commandPayload: string = 'Hello world!';
+
+// Get the has of the command
+let h: string = hash(commandPayload);
+
+// Signing is normally handled by wallets, but if you have the private key
+// available to you, you can also sign in this way:
+let keyPair: KeyPair = {
```
Comment (nillo) [inline packages/libs/cryptography-utils/README.md:30]: IKeyPair

**Review by nillo — APPROVED** [review]

File: packages/libs/cryptography-utils/README.md (hunk 1 above)
Comment (ash-vd) [inline packages/libs/cryptography-utils/README.md:30]:
```suggestion
let keyPair: IKeyPair = {
```

**Comment by Randynamic** [comment]
@Luzzotica can you run a `rush change` https://rushjs.io/pages/commands/rush_change/ so we can merge your PR's.
These commands generate log files for our internal and external releases

**Comment by Luzzotica** [comment]
Done!

**Review by Randynamic — APPROVED** [review]

**Pull request merged**